src/index.html

```diff
@@ -28,7 +28,7 @@
           </li>
 
           <li class="nav__item">
-            <a href="#sampsung" class="nav__link">Samsung</a>
+            <a href="#samsung" class="nav__link">Samsung</a>
```

Comment:
love this

Reply:
need to fix it

Reply:
@ArturErmolenko Do I need to return "sampsung"?

Reply:
nope 😉